crates/runtime/src/traphandlers/backtrace.rs

```diff
@@ -211,7 +212,9 @@ impl Backtrace {
             // The stack grows down, and therefore any frame pointer we are
             // dealing with should be less than the stack pointer on entry
             // to Wasm.
-            assert!(trampoline_sp >= fp, "{trampoline_sp:#x} >= {fp:#x}");
+            // TODO(frank-emrich) Disabled for now, as it does not hold in
+            // presence of continuations.
+            //assert!(trampoline_sp >= fp, "{trampoline_sp:#x} >= {fp:#x}");
```

Comment:
I think this is OK for now. In some (distant) future we'd want some more flexible checking, e.g. if stack-switching feature is off then the assertion is kept.